backend/logging/logging.go

```diff
@@ -2,52 +2,44 @@ package logging
 
 import (
 	"context"
+	"io"
+	"log/slog"
 	"os"
 
-	kitlog "github.com/go-kit/kit/log"
 	"github.com/google/uuid"
 )
 
-// Logger is a generic logging interface. Currently wraps Go-kit's log.Logger
-type Logger kitlog.Logger
```

Comment:
I probably would have kept Logger and just had it wrap *slog.Logger to minimize the differences, but this works too.

Reply:
I originally tried that and ran into some type issues. This ended up being the easiest way to solve it. My hope is that we won't ever need to swap this again so it won't matter but I agree, that would have made it probably simpler.